lib/native-package-installer/platform/ubuntu.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2013  Ruby-GNOME2 Project Team
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+class NativePackageInstaller
+  module Platform
+    class Ubuntu
+      Platform.register(self)
+
+      class << self
+        def current_platform?
+          File.exist?("/etc/lsb-release")
```

Comment:
This is too rough to detect Ubuntu.

Reply:
For Debian, `/etc/debian_version` is used so I think its counterpart is suitable. Please guide me to the right direction. Thanks.

Reply:
`/etc/lsb-release` is generic file. For example, Linux Mint also uses this file.

We should check content when we use `/etc/lsb-release`:

```ruby
return false unless File.exist?("/etc/lsb-release")
File.readlines("/etc/lsb-release").any? do |line|
  line.chomp == "DISTRIB_ID=Ubuntu"
end
```

Reply:
Thanks. 5b82991